plugins/ovdmx/OVDmxPlugin.h

```diff
@@ -0,0 +1,43 @@
+#ifndef PLUGINS_OVDMX_OVDMXPLUGIN_H_
+#define PLUGINS_OVDMX_OVDMXPLUGIN_H_
+
+#include <string>
+#include <vector>
+#include "olad/Plugin.h"
+#include "ola/plugin_id.h"
+
+namespace ola {
+namespace plugin {
+namespace ovdmx {
+
+class OVDmxDevice;
+
+class OVDmxPlugin: public Plugin {
+ public:
+    explicit OVDmxPlugin(PluginAdaptor *plugin_adaptor):
+      Plugin(plugin_adaptor) {
+    }
+
+    std::string Name() const { return PLUGIN_NAME; }
+    std::string Description() const;
+    ola_plugin_id Id() const { return OLA_PLUGIN_EXPERIMENTAL; }
```

Comment:
See the notes here for how plugin IDs get assigned now: https://github.com/OpenLightingProject/ola/blob/master/common/protocol/Ola.proto#L72-L78